src/modules/tags/TagsModal.js

```diff
@@ -43,9 +43,24 @@ const TagsModal = () => {
   const save = async () => {
     if (modalVisible !== (MODALS.SHORTCUT_MODALS.ADD_TAGS_TO_SPOTS)) {
       useMaps.setPointAtCurrentLocation().then(spot => {
-        checkedTagsTemp.map(tag => useTags.addRemoveTagFromSpot(tag, spot));
+        checkedTagsTemp.map(tag => {
+          useTags.addRemoveTagFromSpot(tag, spot);
+        });
       });
     }
+    else {
+      let tagsToUpdate = [];
+      checkedTagsTemp.map(tag => {
+        let spotsListForTagging = [];
+        selectedSpotsForTagging.map(spot => {
+          if (!useTags.tagSpotExists(tag, spot)) spotsListForTagging.push(spot.properties.id);
+        });
+        let tagCopy = JSON.parse(JSON.stringify(tag));
+        tagCopy.spots = tagCopy.spots.concat(spotsListForTagging);
+        tagsToUpdate.push(tagCopy);
+      });
+      useTags.saveTag(tagsToUpdate);
+    }
     else checkedTagsTemp.map(tag => selectedSpotsForTagging.map(spot => useTags.addTagToSpot(tag, spot)));
```

Comment:
Now there are two `else` statements after the `if`

Reply:
.. I don't see the extra else, neither in my code nor in the commit too.

![Screen Shot 2021-02-02 at 3 00 42 PM](https://user-images.githubusercontent.com/28953643/106662319-dbe22580-6567-11eb-95b6-ae66c092199a.png)

Reply:
![image](https://user-images.githubusercontent.com/1493985/106664015-c1a54900-6561-11eb-83fe-dbe50226089c.png)

Reply:
I did a merge into the pull request to resolve a conflict this morning.  I didn't know that would mess that up sorry.

Reply:
Next time there is a conflict I will let you know about it so you can fix it in your branch